### PR TITLE
Translations: Add Turkish and German

### PR DESCRIPTION
**Pull request by Stargazer06**
Hi, these are the Turkish and German translations.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -134,6 +134,16 @@ export const LANG_LIST: LangItem[] = [{
   name: 'Chinese (Traditional)',
   nativeName: '繁體',
   rtl: false,
+  }, {
+  langCode: 'tr',
+  name: 'Turkish',
+  nativeName: 'Türkçe',
+  rtl: false,
+  }, {
```
Comment (mytonwalletorg) [inline src/config.ts:142]:
```suggestion
}, {
```

File: src/config.ts [hunk 2]
```diff
@@ -134,6 +134,16 @@ export const LANG_LIST: LangItem[] = [{
   name: 'Chinese (Traditional)',
   nativeName: '繁體',
   rtl: false,
+  }, {
```
Comment (mytonwalletorg) [inline src/config.ts:137]:
```suggestion
}, {
```

File: src/i18n/de.yaml [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+Sort By Value: Nach Wert sortieren
+Update: Aktualisieren
+Restart: Neustart
+Update MyTonWallet: MeinTonWallet aktualisieren
+Connect Ledger: Ledger verbinden
+Connecting to your Ledger...: Mit Ihrem Ledger verbinden...
+Failed to connect: Verbindung fehlgeschlagen
+Ledger Connected!: Ledger verbunden!
+Select Ledger Wallets: Ledger Wallets auswählen
+"%1$d ausgewählt": %1$d ausgewählt
```
Comment (mytonwalletorg) [inline src/i18n/de.yaml:311]: Why is this key in German?

File: src/i18n/de.yaml [hunk 4]
```diff
@@ -0,0 +1,520 @@
...
+Stake Again: Erneut staken
+Earned: Verdient
+Connect Dapp: Dapp verbinden
+Select wallets to use on this dapp: Wählen Sie Wallet aus, die Sie auf dieser Dapp verwenden möchten.
+Connect: Verbinden
+Dapp Permissions: Dapp-Berechtigungen
+$dapp_can_view_balance: %dappname% wird die Wallet adresse und den Kontostand anzeigen können.
+Send Transaction: Transaktion senden
+Payload: Payload
+$many_transactions: %1$d Transaktionen
```
Comment (mytonwalletorg) [inline src/i18n/de.yaml:270]:
```suggestion
$many_transactions: "%1$d Transaktionen"

```

Comment (mytonwalletorg) [reply src/i18n/de.yaml:270]: Strings that start from `%` should be wrapped in quotes `"`.

**Review by mytonwalletorg — CHANGES_REQUESTED** [review]

File: src/i18n/de.yaml [hunk 5]
```diff
@@ -0,0 +1,520 @@
...
+Enter code or use Touch ID: Code eingeben oder Touch ID verwenden
+Enter code or use biometrics: Code eingeben oder Biometrie verwenden
+Wrong code, please try again: Falscher Code, bitte versuchen Sie es erneut.
+Scan your fingerprint: Legen Sie Ihren Fingerabdruck auf.
+Incorrect code, please try again.: Falscher Code, bitte versuchen Sie es erneut.
+Correct: Korrekt
+Confirm Operation: Operation bestätigen
+Confirm Swap: Swap bestätigen
+"%amount% to %address%": "%amount% zu %address%"
+"%amount_from% to %amount_to%": "%amount_from% zu %amount_to%""
```
Comment (mytonwalletorg) [inline src/i18n/de.yaml:512]:
```suggestion
"%amount_from% to %amount_to%": "%amount_from% zu %amount_to%"

```

File: src/i18n/tr.yaml [hunk 6]
```diff
@@ -0,0 +1,518 @@
...
+Cancel: İptal
+Insecure Password: Güvenli olmayan şifre
+Your have entered an insecure password, which can be easily guessed by scammers.: Dolandırıcılar tarafından kolayca tahmin edilebilecek güvenli olmayan bir şifre girdiniz.
+Continue or change password to something more secure?: Devam edilsin mi yoksa şifre daha güvenli bir şifreyle değiştirilsin mi?
+Change: Değiştir
+"%1$d Secret Words": "%1$d Gizli Kelime"
+$auth_import_mnemonic_description: |
+  Cüzdanı oluştururken yazdığınız %1$d gizli kelimeyi girerek cüzdanınıza erişimi geri kazanabilirsiniz.
+Let’s make sure your secrets words are recorded correctly.: Gizli kelimelerinizin doğru kaydedildiğinden emin olun.
+$mnemonic_check_words_list: **%1$** numaralı kelimeleri aşağıya girin.
```
Comment (mytonwalletorg) [inline src/i18n/tr.yaml:33]:
```suggestion
$mnemonic_check_words_list: "**%1$** numaralı kelimeleri aşağıya girin."
```

File: src/i18n/tr.yaml [hunk 7]
```diff
@@ -0,0 +1,518 @@
...
+Logo: Logo
+$about_description1: Kriptoyu güvenle saklayın, merkeziyetsiz uygulamaları keşfedin ve **ışık hızında** blok zinciri ödemeleri yapın.
+$about_description2: Cüzdan **non-custodial ve güvenli** bir cüzdandır. Geliştiricilerin paranıza, tarayıcı geçmişinize veya diğer bilgilerinize erişimi ** yoktur **.
+What is TON Proxy?: TON Proxy nedir?
+$about_extension_description1: TON Proxy, TON Sitelerine anonim şekilde erişmeolanağı sağlayarak **merkeziyetsizleştirilmiş internetin** yolunu açar.
+More info and demo.: Daha fazla bilgi ve demo.
+$about_extension_description2: TON Magic, resmi Telegram Web uygulaması için sunduğu yamayla **Telegram entegrasyonu** sağlar.
+$about_extension_description3: Herhangi bir Telegram kullanıcısına TON göndermek ve almak için bu özelliği etkinleştirin.
+$about_proxy_magic_title: Peki ya %ninja% TON Proxy ve %unicorn% TON Magic nedir?
+$about_proxy_magic_description: **Merkeziyetsiz internetin** keyfini çıkarmak, **TON Sitelerine** erişmek ve **Telegram ile entegrasyonun** faydalarından yararlanmak için %extension_link%'i yükleyebilirsiniz.
```
Comment (mytonwalletorg) [inline src/i18n/tr.yaml:73]:
```suggestion
$about_proxy_magic_description: "**Merkeziyetsiz internetin** keyfini çıkarmak, **TON Sitelerine** erişmek ve **Telegram ile entegrasyonun** faydalarından yararlanmak için %extension_link%'i yükleyebilirsiniz."
```

File: src/i18n/tr.yaml [hunk 8]
```diff
@@ -0,0 +1,518 @@
...
+Name: İsim
+Save: Kaydet
+Copy: Kopyala
+Remove From Saved Addresses: Kayıtlı adreslerden kaldır
+Add To Saved Addresses: Kayıtlı adreslere ekle
+View Address on TON Explorer: TON Explorer'da adresi görüntüle
+No suggestions, you're on your own!: Önerimiz bulunmuyor, tek başınasınız!
+Play Sounds: Sesleri çal
+Focus on asset value rather than current balance: Mevcut bakiyeden ziyade varlık değerine odaklanır
+$logout_confirm: **Tüm** cüzdanları kaldırın
```
Comment (mytonwalletorg) [inline src/i18n/tr.yaml:196]:
```suggestion
$logout_confirm: "**Tüm** cüzdanları kaldırın"
```

File: src/i18n/tr.yaml [hunk 9]
```diff
@@ -0,0 +1,518 @@
...
+  MyTonWallet, uygulama ile yalnızca bir etkin sekmeyi destekler.
+  Bu sekmeyi kullanmaya devam etmek için lütfen sayfayı yeniden yükleyin ya da kapatın.
+Reload App: Uygulamayı yeniden yükleyin
+Please confirm operation on your Ledger: Lütfen Ledger'ınızdaki işlemi onaylayın
+Token Not Found: Token bulunamadı
+Try another keyword or address.: Başka bir anahtar kelime veya adres deneyin.
+Decrypt: Şifre çöz
+Encrypted Message: Şifrelenmiş Mesaj
+Message is encrypted.: Mesaj şifrelenmiştir.
+$dapp_ledger_warning1: **Ledger** cüzdanınızı kullanarak çok yönlü bir işlem göndermek üzeresiniz. İlgili her işlemi **tek tek manuel olarak** imzalamanız gerekecektir.
```
Comment (mytonwalletorg) [inline src/i18n/tr.yaml:338]:
```suggestion
$dapp_ledger_warning1: "**Ledger** cüzdanınızı kullanarak çok yönlü bir işlem göndermek üzeresiniz. İlgili her işlemi **tek tek manuel olarak** imzalamanız gerekecektir."
```

Comment (Stargazer06) [reply src/i18n/tr.yaml]: I was not informed about the rule about [**... → `: "**..."].   (As a translator I have limited knowledge about coding spelling rules). Are you going to edit by yourself, or would you like me to handle the editing of the rest of the file?

**Pull request merged**